### PR TITLE
Request to merge Milestone 5 branch to default branch

### PR DESCRIPTION
**Pull request by mikenath223**
Implementations:
- Add state to App component.
- Add clickHandle to ButtonPanel Component.
- Create handleClick event to the Button Component.

Kindly review.
Thanks.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const x = Big(num1);
+  const y = Big(num2);
+
+  if (operand === '+') {
+    return x.plus(y).toString();
+  } if (operand === '-') {
+    return x.minus(y).toString();
+  } if (operand === '÷') {
+    return x.div(y).toString();
+  } if (operand === 'x') {
```
Comment (shloch) [inline src/logic/operate.js:13]: You didn't manage DIVISION BY ZERO.........return something like `INFINITY` on screen or some other form of error management

Screen Shot 2020-04-06 at 6 50 25 PM

Comment (mikenath223) [reply src/logic/operate.js:13]: Thanks for this. I will fix this.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -3,13 +3,17 @@ import PropTypes from 'prop-types';
...
-    const { onClick } = this.props;
     return (
       <Button
         classname={c}
         name={n}
-        onClick={() => onClick(n)}
+        clickHandle={n => this.clickHandler(n)}
         wide={width}
         color={col}
       />
```
[the remark this answers is not in the document]
Comment (mikenath223) [reply src/components/ButtonPanel.js:19]: Thanks.

**Comment by mikenath223** [comment]
Thanks a lot. Applying fix now.

**Comment by mikenath223** [comment]
Thanks for the screencast.

**Pull request merged**